provider_shopper/pubspec.yaml

```diff
@@ -13,6 +13,13 @@ dependencies:
   # Import the provider package.
   provider: ^5.0.0
 
+  # plugin is not yet part of the flutter framework
+  window_size:
+    git:
+      url: https://github.com/google/flutter-desktop-embedding.git
+      path: plugins/window_size
+      ref: 03d957e8b5c99fc83cd4a781031b154ab3de8753
```

Comment:
Any particular reason to pin a git ref?

Reply:
Yes, there's a reason. This package is yet not part of the flutter framework nor it is available on pub.dev. The install instructions are here: https://github.com/google/flutter-desktop-embedding/blob/master/plugins/README.md

Reply:
Yes, I understand the need for git dependency. I'm questioning pinning to a specific git hash. As long as the `pubspec.lock` is checked in, that should be enough to replicate builds.